nest-city-account/src/admin/subservices/admin-cron.subservice.ts

```diff
@@ -188,9 +188,21 @@ export class AdminCronSubservice {
     }
     this.logger.log(`${COGNITO_SYNC_CONFIG_DB_KEY} turned ON, starting`)
 
-    const result = await this.adminService.syncCognitoToDb()
+    try {
+      await this.adminService.syncCognitoToDb()
+    } catch (error) {
+      this.logger.error(error)
+      this.logger.log(
+        `syncCognitoToDb unsuccessfull, change ${COGNITO_SYNC_CONFIG_DB_KEY} to false.`
+      )
+
+      await this.prismaService.config.update({
+        where: { key: COGNITO_SYNC_CONFIG_DB_KEY },
+        data: { value: { active: false } },
+      })
```

Comment:
This can't be here. You whatever you set here, is overridden by the value you set after the try. You are setting both to false, so keep only the one after the try-catch block.